person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rents(rent)
+    @rents.push(rent)
+    rent.person = self
+  end
```

Comment:
- [ ] Let's kindly check out the comment above ⬆️ and modify this method accordingly